Geometry/TrackerGeometryBuilder/test/GeoHierarchyTest.cc

```diff
@@ -40,19 +40,22 @@
 #include "Geometry/TrackerGeometryBuilder/interface/trackerHierarchy.h"
 
 #include "DataFormats/Common/interface/Trie.h"
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
 
 
 #include<string>
 #include<iostream>
 
 template<typename Det>
 struct Print {
+  const TrackerTopology* m_tTopo;
+  explicit Print(const TrackerTopology* tTopo) : m_tTopo(tTopo) {}
   // typedef edm::TrieNode<Det> const node;
   void operator()(Det det, std::string const & label) const {
     if (!det) return; 
     for (size_t i=0; i<label.size();++i)
       std::cout << int(label[i]) <<'/';
-    std::cout << " " << det->geographicalId() << std::endl;
+    std::cout << " " << m_tTopo->print(det->geographicalId()) << std::endl;
```

Comment:
as I wrote on slack yesterday, I think this is just meant to print out the uint32_t, so I think we should write `det->geographicalId().rawId()` instead (and remove the references to `TrackerTopology` from `Print`